src/Akeneo/Connectivity/Connection/front/tests/src/connect/pages/MarketplacePage.test.tsx

```diff
@@ -21,7 +21,7 @@ jest.mock('@src/connect/components/Marketplace', () => ({
     Marketplace: jest.fn().mockImplementation(() => null),
 }));
 
-test('The marketplace page display the developer mode tag when enabled', async () => {
+test('The marketplace page display the developer mode tag and the "create a test app" button when enabled', async () => {
```

Comment:
```suggestion
test('The marketplace page display the developer mode when enabled', async () => {
```

I think we can reduce the notion of "developer mode tag" + "create test app button" under "the developer mode"